challenges/js-array-methods/js-array-methods_filter/index.js

```diff
@@ -23,13 +23,19 @@ const cards = [
   },
 ];
 
-const onlyCardWithIdTwo = null;
+const onlyCardWithIdTwo = cards.filter((card) => card.id === "2");
 
-const allCardsWith3Tags = null;
+const allCardsWith3Tags = cards.filter((card) => card.tags.length === 3);
 
-const allCardsThatAreNotBookmarked = null;
+const allCardsThatAreNotBookmarked = cards.filter(
+  (card) => card.isBookmarked === false || card.isBookmarked === undefined
+);
```

Comment:
Ha! smart solution!

Reply:
It's actually very verbose and does not capture the case where `isBookmarked` would be equal to `null` so I think a better solution would be to simply evaluate for falsy values by doing
```suggestion
  (card) => !card.isBookmarked;
);
```